setup.py

```diff
@@ -89,6 +87,7 @@ def parse_readme(readme: str) -> str:
 # packages dependencies are changed.
 extras_require = {
     "sky": ["skypilot[docker]==0.4.1"],
+    "data": ["pandas", "pyarrow"],
```

Comment:
@dongreenberg should `datasets` and `dask` be included here too?

Reply:
No I think this is fine.